protocol/app/upgrades/v0.2.2/upgrades.go

```diff
@@ -0,0 +1,160 @@
+package v0_2_2
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	"github.com/dydxprotocol/v4-chain/protocol/lib/metrics"
+	clobmodulekeeper "github.com/dydxprotocol/v4-chain/protocol/x/clob/keeper"
+	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+)
+
+func CreateUpgradeHandler(
+	mm *module.Manager,
+	configurator module.Configurator,
+	clobKeeper clobmodulekeeper.Keeper,
+) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		ctx.Logger().Info("Running v0.2.2 hard fork...")
+
+		// Update clob pairs.
+		clobPairs := clobKeeper.GetAllClobPairs(ctx)
+		for _, clobPair := range clobPairs {
+			switch clobPair.Id {
+			case 0:
```

Comment:
@jayy04 can you find someone who can double check these numbers?

Reply:
[Confirmed](https://dydx-team.slack.com/archives/C03FP255RRU/p1694655414499069?thread_ts=1694644218.746589&channel=C03FP255RRU&message_ts=1694655414.499069) by Yu Jin